diff -up libxml2-2.9.9/python/setup.py.in.omv~ libxml2-2.9.9/python/setup.py.in
--- libxml2-2.9.9/python/setup.py.in.omv~	2019-10-20 01:29:53.003584609 +0200
+++ libxml2-2.9.9/python/setup.py.in	2019-10-20 01:30:01.736624520 +0200
@@ -85,7 +85,7 @@ if iconv_includes == "":
 
 # those are added in the linker search path for libraries
 libdirs = [
-os.path.join(ROOT,'lib'),
+# os.path.join(ROOT,'lib'), -L/usr/lib is EVIL
 ]
 
 xml_files = ["libxml2-api.xml", "libxml2-python-api.xml",
